symphony-bdk-examples/bdk-core-examples/src/main/java/com/symphony/bdk/examples/UserExampleMain.java

```diff
@@ -31,5 +36,10 @@ public static void main(String[] args) throws Exception {
     log.info("Account type: {}", userDetailList.get(0).getUserAttributes().getAccountType());
     log.info("Company: {}", userDetailList.get(0).getUserAttributes().getCompanyName());
     log.info("Role: {}", userDetailList.get(0).getRoles());
+
+    AuthSession oboSession = bdk.obo(Obo.username("hong.le"));
+    StreamFilter streamFilter = new StreamFilter().addStreamTypesItem(new StreamType().type(StreamType.TypeEnum.IM));
+    List<StreamAttributes> streamsList = bdk.streams().listStreams(streamFilter, oboSession);
```

Comment:
how would we handle session expiration here?

Reply:
when the session expiration, the API call returns error 401, in this case, we will refresh the session by calling `AuthSession#refresh`